concordium-contracts-common/src/schema.rs

```diff
@@ -8,6 +8,8 @@ use crate::{impls::*, traits::*, types::*};
 use alloc::boxed::Box;
 #[cfg(not(feature = "std"))]
 use alloc::{collections, string::String, vec::Vec};
+#[allow(unused_imports)]
```

Comment:
We don't do that here :) 

Please only import it in the context that it is needed in, so that allow(unused_imports) is not necerssary.

Reply:
Around line 1016 in the inner module.

Reply:
ah, that makes sense.. couldn't figure out why it complained - thanks!